docs/package.json

```diff
@@ -55,7 +55,7 @@
     "lodash": "^4.17.21",
     "lz-string": "^1.4.4",
     "markdown-to-jsx": "^7.1.5",
-    "marked": "^4.0.6",
+    "marked": "^3.0.8",
```

Comment:
The monorepo was using the version installed by Typedoc
Now that we removed it, we must keep our version sync with the one in the core instead of the 4.X (which was never used I think).